lib/codemod/package.json

```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "git+https://github.com/storybooks/storybook.git"
   },
-  "main": "src/index.js",
+  "main": "dist/index.js",
```

Comment:
This package is never ever required by anything. This explains why this bug was never found.